core/src/utils/test/hardware-back-button.spec.ts

```diff
@@ -1,11 +1,12 @@
+import type { BackButtonEvent } from '../../../src/interface';
 import { startHardwareBackButton } from '../hardware-back-button';
 
 describe('Hardware Back Button', () => {
   beforeEach(() => startHardwareBackButton());
   it('should call handler', () => {
     const cbSpy = jest.fn();
     document.addEventListener('ionBackButton', (ev) => {
```

Comment:
Why not set the event type here?

```suggestion
    document.addEventListener('ionBackButton', (ev: BackButtonEvent) => {
```

Reply:
You can't. It'll result in a typing error since `BackButtonEvent` does not match the type signature that comes from `.addEventListener`.

Reply:
FWIW this is why we have this logic for this custom type: https://github.com/ionic-team/ionic-framework/blob/f143bd0a11941739ee616908891509b48bd48463/core/src/utils/browser/index.ts#L30-L41

But we don't get that when using the `Window` global type.